src/coreclr/vm/interpreter.h

```diff
@@ -914,6 +914,16 @@ class Interpreter
 
     void*      GetParamTypeArg() { return m_genericsCtxtArg; }
 
+    // Also see namedintrinsiclist.h
+    enum InterpreterNamedIntrinsics : unsigned short
```

Comment:
We can move `namedintrinsiclist.h` to `src/native/minipal` and reuse it like `#include <minipal/namedintrinsiclist.h>` in JIT and interpreter (these include paths are already in place).

ps - it will also help future code unification efforts (e.g. https://github.com/dotnet/runtime/pull/91993).

Reply:
I do not think it makes sense to unify the strategy for handling intrinsics in the JIT with the strategy for handling intrinsics in the interpreter. There are too many of them. If we were to make the interpreter a production quality component, I think we would want to depend on the JIT or AOT compiled code to handle most of the intrinsics.

Reply:
I think that doing it in a way that makes it so that intrinsics don't have to be implemented twice for both JIT and interpreter would be best here.